### DISCUSSION TITLE
Some Question

### DISCUSSION THREAD

**Discussion by Doc-LI-desgin** [spm/spm#41] [General]
I downloaded and ran SPMv7771 on matlab2018b, but I found that my SPM interface was not divided into specify 1st level and specify 2nd level, but there are two options for Basic model and Bayesian, why is this, and after I do a two-sample t-test, I find that I don't have the option of FDR correction, why is that? The following screenshot of the interface is provided
<img width="350" alt="微信图片_20241114183147" src="https://github.com/user-attachments/assets/7e2735cc-b91b-4f04-8308-c5f8d4d33843">

**Comment by SPMcentral** [comment]
You have started SPM in PET mode. Try starting it in fMRI mode, by typing: `spm fmri` and pressing enter , or in M/EEG mode, by typing: `spm eeg` and pressing enter.

Best
Peter
